config/default-spinnaker-local.yml

```diff
@@ -96,7 +96,6 @@ providers:
       # These credentials use authentication information at ~/.kube/config
       # by default.
       name: my-kubernetes-account
-      namespace: default
```

Comment:
This is a useless default since it's never consumed in `clouddriver.yml`, it overrides whats in `.kube/config`, and is already the default if nothing is specified in `.kube/config`. So I removed it.